addons/udes_stock/models/stock_picking.py

```diff
@@ -393,3 +406,48 @@ def _create_move(self, move_values):
         """
         Move = self.env["stock.move"]
         return Move.create(move_values)
+
+    def find_empty_pickings_to_unlink(self):
```

Comment:
Do we need this method for the push from drop functionality?

Reply:
See below comment.